src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx

```diff
@@ -265,10 +295,19 @@ class Vivify extends Analyzer {
 
   private _tallyCastEntry(event: CastEvent) {
     const vivifyHits = getVivifiesPerCast(event) as HealEvent[];
+    const invigoratingMistHits = vivifyHits.filter(
+      (invigMists) => invigMists.ability.guid === SPELLS.INVIGORATING_MISTS_HEAL.id,
+    );
     let vivifyGoodCrits = 0;
     let vivifyWastedCrits = 0;
+    let fullOverhealHits = 0;
+    let healingPerCast = 0;
+    let overhealPerCast = 0;
     if (this.upliftedSpirits.active) {
       vivifyHits.forEach((event) => {
+        const effective = event.amount + (event.absorbed || 0);
+        healingPerCast += effective;
```

Comment:
Seems like an unnecessary store
```suggestion
        healingPerCast += event.amount + (event.absorbed || 0);
```

Reply:
Just readability for line 325